src/MetalPerformanceShaders/MPSDefs.cs

```diff
@@ -81,10 +83,19 @@ public struct MPSScaleTransform {
 	}
 
 	// MPSImageHistogram.h
+	[StructLayout (LayoutKind.Explicit)]
 	public struct MPSImageHistogramInfo {
+		[FieldOffset (0)]
 		public nuint NumberOfHistogramEntries;
+#if XAMCORE_2_0
+		[FieldOffset (8)]
+#else
+		[FieldOffset (4)]
+#endif
```

Comment:
Oops, that's wrong, it's supposed be a 32-bit/64-bit ifdef, not Classic/Unified.